### PR TITLE
Feature achievements

### LINKED ISSUE #123: Picture-unlock of all achievements instead of one.
In the Achievementscreen controller I set each picture to check whether it's corresponding achievements has been achieved or not. When it has been achieved, the picture should be shown, else it is white.
However, as soon as one achievement is gained, all pictures are shown. I do not understand why, could anybody take a look and guide me to fix this bug?

(Should we also use octopull-tags in issues?)
<bug

### PR DESCRIPTION
**Pull request by jessearens**
This pull request performs two things:
1. It somewhat improves the code readability of the AchievementScreenController by removing some unnecessary duplicate code. Some checkstyle errors were also found and fixed. Next up would be to redesign this part of the controller to support the bigger and smaller achievement sprites and to code this so that it is even better (probably going to use gridpanes). 
2. It fixes the bug of issue #123. The bug was caused by an incorrect placement of the colorAdjust object within the AchievementScreenController.

Fixes #123

<feature,bug

### CODE REVIEW

**Comment by Taeir** [comment]
Looks good! I see that there are still some unused fields, but I assume that those will be used soon, so I think this is good to be merged.

<feature,bug

**Comment by jessearens** [comment]
Correct, they will be used as soon as I get to implementing these next achievements. Preferably this sprint.

<feature,bug

**Comment by jessearens** [comment]
I decided to continue on working on the code, even though this pull request wan't merged yet.

Please bear in mind that 1,400 of the LOC is the implementation of the new FXML screen. A look through the other classes that changed is suggested first. Looking at the implementation of the FXML screen might be easiest through the SceneBuilder application.

<review,feature,user_interface

**Comment by Diocruel** [comment]
It looks good. The unused fields are anoying though. You might wanna space out the achievements a bit more; are we really gonna have more than 10-20 achievements? Let's not lose too much time on this! Good job so far. 

<review, gui

File: src/main/java/com/github/fishio/control/AchievementScreenController.java [hunk 1]
```diff
@@ -21,8 +21,11 @@
 public class AchievementScreenController implements ScreenController {
 	
 	private Log log = Log.getLogger();
-	
-	
+	private Achievement playerDeath = new Achievement("playerDeath");
+	private Achievement enemyKill = new Achievement("enemyKill");
+	private Achievement hitWall = new Achievement("HitWall");
+	private Achievement revive = new Achievement("Revive");
+	private Achievement pun = new Achievement("Pun");
```
Comment (ChielBruin) [inline src/main/java/com/github/fishio/control/AchievementScreenController.java]: Nice layout! :+1: 
<layout

File: src/main/java/com/github/fishio/control/AchievementScreenController2.java [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+	private Log log = Log.getLogger();
+	private Achievement playerDeath = new Achievement("playerDeath");
+	private Achievement enemyKill = new Achievement("enemyKill");
+	private Achievement hitWall = new Achievement("HitWall");
+	private Achievement revive = new Achievement("Revive");
+	private Achievement pun = new Achievement("Pun");
+
+	@FXML
+	private Button btnBackToMenu;
+	private Label bigachieve1;
```
Comment (Taeir) [inline src/main/java/com/github/fishio/control/AchievementScreenController2.java]: I'm afraid all these fields need a `@FXML` to be useful. Besides, we don't really need them in the code, they are not used anyway.

<naming

Comment (jessearens) [inline src/main/java/com/github/fishio/control/AchievementScreenController2.java]: Correct, they will all need a @FXML annotation. Right now they are not used, but the next commit would make them used. The idea is that every picture has to be set on or off depending on whether their achievement is gained or not.

To be honest, I did not intend to add  ea41f25 and ea5dc06 to this pull request yet. However, I thought the pull request was already merged and therefore I pushed these two commits. I think for now it's best to just wait for my next commit which will actually replace the old Achievement screen with the next one.

<naming,review

**Comment by jessearens** [comment]
Now the pull request can be looked into again and if there are no more comments, merged.

<review

**Comment by ChielBruin** [comment]
It looks good!
After you get it all working, you might want to refactor the code to remove the lists of almost identical code.

<feature

**Comment by ChielBruin** [comment]
It all looks good!
<review

**Comment by Taeir** [comment]
You also might want to remove all the commented out code.
For the rest, the changes look good.

<feature

**Comment by jessearens** [comment]
Removed the commented code.

<layout,feature

**Pull request merged**